### PR TITLE
Move Paper and Fastutil imports to fully qualified names where possible

### PR DESCRIPTION
**Pull request by broccolai**
Now that every class isn't in `net.minecraft.server` many imports can be removed

### CODE REVIEW

File: Spigot-Server-Patches/0004-MC-Utils.patch [hunk 1]
```diff
@@ -3657,15 +3641,24 @@ index b88b0890cb5b5b0c4c2c35456fdffb02e8cfcf91..bc580b35bdffddc5675a7bd508010575
...
      private final MinecraftServer server;
      public final WorldDataServer worldDataServer; // CraftBukkit - type
-@@ -1687,7 +1688,7 @@ public class WorldServer extends World implements GeneratorAccessSeed {
+@@ -1687,7 +1687,7 @@ public class WorldServer extends World implements GeneratorAccessSeed {
                  ObjectIterator objectiterator = spawnercreature_d.b().object2IntEntrySet().iterator();
  
                  while (objectiterator.hasNext()) {
 -                    it.unimi.dsi.fastutil.objects.Object2IntMap.Entry<EnumCreatureType> it_unimi_dsi_fastutil_objects_object2intmap_entry = (it.unimi.dsi.fastutil.objects.Object2IntMap.Entry) objectiterator.next();
-+                    Object2IntMap.Entry<EnumCreatureType> it_unimi_dsi_fastutil_objects_object2intmap_entry = (Object2IntMap.Entry) objectiterator.next(); // Paper - decompile fix
++                    it.unimi.dsi.fastutil.objects.Object2IntMap.Entry<EnumCreatureType> it_unimi_dsi_fastutil_objects_object2intmap_entry = (it.unimi.dsi.fastutil.objects.Object2IntMap.Entry) objectiterator.next(); // Paper - decompile fix
```
Comment (jpenilla) [inline Spigot-Server-Patches/0004-MC-Utils.patch]: Get rid of the diff here, it's only adding a `// Paper` comment now

File: Spigot-Server-Patches/0010-Adventure.patch [hunk 2]
```diff
@@ -1365,33 +1317,25 @@ index db155011112aba2b1e3ec5b337db25dfb7b511fb..c417250f66805788d63434e2c360349a
...
- import java.io.File;
- import java.net.SocketAddress;
- import java.text.SimpleDateFormat;
-@@ -92,6 +93,7 @@ import org.apache.logging.log4j.LogManager;
+@@ -92,6 +92,7 @@ import org.apache.logging.log4j.LogManager;
  import org.apache.logging.log4j.Logger;
  
  // CraftBukkit start
-+import io.papermc.paper.adventure.PaperAdventure; // Paper
++import io.papermc.paper.adventure.PaperAdventure;
```
Comment (jpenilla) [inline Spigot-Server-Patches/0010-Adventure.patch]: This import can be removed

**Review by jpenilla — CHANGES_REQUESTED** [review]
In general, looks good. Seems to have covered pretty much all of the now-unnecessary `io.` and `it.` imports. In reviewing this, I did notice quite a few `net.minecraft.` and `org.bukkit.` imports that could probably be replaced will fully qualified names as well, however those would probably be best addressed in a separate PR.

**Review by jpenilla — APPROVED** [review]

**Review by Proximyst — CHANGES_REQUESTED** [review]
I don't think this is a good change: we've settled on a decision to just let the IDE handle imports prior, and I see no reason for us to change that now, especially as the imports section is now actually readable.

Blocking on discussion. Requires core approval.

**Comment by electronicboy** [comment]
I sallute the effort but I don't see any pressing push for this, maybe on a major release but we've already got a queue we're tryna work through, such a commit would impact a good chunk of them

**Review by kashike — APPROVED** [review]
Approving in general - can delay merging this for a while, if we want.

**Comment by JRoy** [comment]
Closing as per https://discord.com/channels/289587909051416579/555469074080202765/1002597798153293824

**Pull request closed without merging**